### DISCUSSION TITLE
Foreground services

### DISCUSSION THREAD

**Discussion by ishanj-lk** [tauri-apps/tauri#11896] [Q&A]
Will there be functionality like foreground services in the future, especially for notifications?

**Comment by FabianLars** [comment]
Very likely. There's hasn't been made _any_ further planning but it's not exactly uncommon so it will be implemented sooner or later. (btw, you have full access to the swift and kotlin projects so you can always just do it the native way)

**Reply by Vexcited** [reply]
Did something similar where I made a Tauri plugin that I use as backend for my app.

It allows me to implement a foreground service like this, just like the OP asked : https://github.com/Vexcited/StayReal/blob/main/internal-api/android/src/main/java/NotificationService.kt

**Reply by ishanj-lk** [reply]
@Vexcited Thank you for the information 🙂
